projects/systelab-components/src/lib/contextmenu/abstract-context-menu.component.ts

```diff
@@ -11,6 +11,10 @@ export abstract class AbstractContextMenuComponent<T> extends AbstractContextCom
 	@Output() public action = new EventEmitter();
 
 	public _contextMenuOptions: Array<T>;
+	protected previousShownMenu: Array<string> = new Array<string>();
```

Comment:
protected previousShownMenu: Array<string> = [];

Do not set the type twice

Reply:
OK